runtime/tuple.go

```diff
@@ -75,6 +75,14 @@ func tupleContains(f *Frame, t, v *Object) (*Object, *BaseException) {
 	return seqContains(f, t, v)
 }
 
+func tupleCount(f *Frame, args Args, kwargs KWArgs) (*Object, *BaseException) {
+	argc := len(args)
+	if argc != 2 {
+		return nil, f.RaiseType(TypeErrorType, fmt.Sprintf("count() takes exactly one argument (%d given)", argc))
```

Comment:
I did the hand-coded error check so that I could match the CPython error message.  Plus, isn't the type error already caught?:

```
$ cat foo.py 
print tuple.count("ab", 2)
$ ./build/bin/grumpc foo.py > foo.go
$ go run foo.go                     
TypeError: unbound method count() must be called with tuple instance as first argument (got str instance instead)
exit status 1
```

Also, I did the exact same style check in `strCount` and `listCount`.  I would prefer to do it the same way everywhere.